web/src/Router.re

```diff
@@ -0,0 +1,28 @@
+type t =
+  | Home
+  | New
+  | Stats(string)
+  | Notfound;
+
+let unwrap =
+  fun
+  | Home => "home"
+  | New => "new"
+  | Stats(_) => "stats"
+  | _ => "404";
+
+let parseUrl = (url: ReasonReactRouter.url) =>
+  switch (url.path) {
+  | ["/"] => Home
+  | ["/new"] => New
+  | ["/stats", slug] => Stats(slug)
+  | _ => Notfound
+  };
+
+/*
+  use for handle button or ahref click
+ */
+let linkTo = (path, event) => {
```

Comment:
I don't see this used anywhere?

Reply:
e:g 

``` 
<button onClick={event => Home->Router.unwrap->Router.linkto(event)}  />
```

Reply:
any conflict with this?

```suggestion
let make = (~href, ~className="", ~style: ReactDOMRe.Style.t=?, ~children) => {
  let handleClick = (href, e) => {
    ReactEvent.Mouse.preventDefault(e);
    ReasonReactRouter.push("/#" ++ href);
  };

  <a href={"/#" ++ href} style className onClick={e => handleClick(href, e)}>
    children
  </a>;
};
```

https://github.com/evilfactorylabs/gow/blob/master/web/src/components/Link.re

Reply:
I think not conflict, as `linkTo` just functional and `Link` is component based :D

Reply:
OK thanks, let's see where we can use `linkTo`method. thank you!